README.md

```diff
@@ -129,6 +128,8 @@ After you have the application running, here are some places to explore:
 
 Use the standard Rails test commands: `rails test`, `rails test:system`, etc.
 
+Note, in order to get system tests to run, you will need `chromedriver` installed. See [Requirements section](#requirements) above.
+
```

Comment:
I thought it may be useful to move the whole `requirements` header to be a subsection of the `development` header, but I didn't know if that would be too far-reaching. Let me know if you're open to seeing what that would look like.

Reply:
That sounds good to me! If you want to make that change in this PR, I can wait to merge it.